org.eclipse.xpect.releng/target-platforms/eclipse_2024_06-xtext_nightly/org.eclipse.xpect.target.eclipse_2023_09-xtext_nightly.target

```diff
@@ -34,7 +34,7 @@
 <unit id="org.junit" version="4.13.2.v20230809-1000"/>
 <unit id="org.junit.source" version="4.13.2.v20230809-1000"/>
 <unit id="io.github.classgraph" version="0.0.0"/>
-<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09"/>
+<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2024-12"/>
```

Comment:
am not sure if this will work.
i assume we have the old one intentionally.
so id expect a 2nd entry with org.apache.commons.commons-logging in the body